administration/src/bp-modules/cards/hooks/useCardGenerator.ts

```diff
@@ -49,6 +49,7 @@ const useCardGenerator = (region: Region) => {
   })
   const [cardBlueprints, setCardBlueprints] = useState<CardBlueprint[]>([])
   const [state, setState] = useState(CardActivationState.input)
+  const [applicationIdToMarkAsProcessed, setApplicationIdToMarkAsProcessed] = useState<number>()
```

Comment:
```suggestion
  const [applicationIdToMarkAsProcessed, setApplicationIdToMarkAsProcessed] = useState<number | undefined>()
```
If it isn't initialized, shouldn't it also include `undefined`

Reply:
sorry i didn't see that same issue was already resolved. this can be ignored